plugins/kubernetes/src/components/Cluster/Cluster.tsx

```diff
@@ -104,9 +105,14 @@ type ClusterProps = {
   clusterObjects: ClusterObjects;
   podsWithErrors: Set<string>;
   children?: React.ReactNode;
+  customPodTableColumns?: TableColumn<V1Pod>[];
 };
 
-export const Cluster = ({ clusterObjects, podsWithErrors }: ClusterProps) => {
+export const Cluster = ({
+  clusterObjects,
+  podsWithErrors,
+  customPodTableColumns,
```

Comment:
Could we wrap customPodTableColumns in a Customisation object which contains info like this?

This way if we want to allow something else to be customisable we don't have to add it to all the component constructors again.

Then maybe replace it with a react context or something